### DISCUSSION TITLE
ESPRESSO

### DISCUSSION THREAD

**Discussion by mjzahn** [ECCO-Hackweek/ecco-2024#158] [General]
Exploring transport in the Southern Ocean using EMU to investigate model sensitivities to forcing perturbations

**Comment by mjzahn** [comment]
*Summary for work completed on Oct 15, 2024*
- Developed concrete project plan for team
- Created masks for EMU runs, one across the Drake Passage and another that included two points at either end
- Generated preliminary plots of SSH and horizontal transport in OSS
- Initiated adjoint model run

**Reply by noahrosen** [reply]
To do:

- [x]  Run 3 months of adjoint 
- [x]  Run sampling tool 
- [ ] Run attribution tool
- [ ] Run convolution tool
- [ ] Plot adjoint results
- [ ] Plot convolution results
- [ ] Plot sampling results
- [ ] Plot attribution results

**Reply by ojhoundegnonto** [reply]
Our group is focused on Exploring transport in the Southern Ocean using EMU to investigate model sensitivities to forcing perturbations through the Drake Passage. So far, we:

- Ran adjoint tool for 12 months at the Drake Passage
- Ran Sampling and Attribution tools
- Made some plots of results

Examples of the Attribution tool:
![Atr_01](https://github.com/user-attachments/assets/a9c96850-b684-486d-bf4c-1c1bc7d35c02)
![Attr_tool_at_Drake_Passage](https://github.com/user-attachments/assets/ee4395d3-427d-4796-972d-3d2818138927)

**Comment by noahrosen** [comment]
Summary for work completed on Oct 16, 2024

- Ran adjoint on Drake Passage mask for 12 months
- Ran attribution and convolution
- Plots of sensitivity to wind stress forcing in agreement with our expectations 
- Obtained adjoint results for full drake passage transport mask (from Ichiro)

**Comment by mjzahn** [comment]
*Summary for work completed on Oct 17, 2024*

- Generated figures for final poster, including results from attribution, convolution, and adjoint
- Finished full draft of poster
- Made animation of adjoint sensitivities in southern ocean
- Learned how to collaborate on github repo hosted on the ECCO page
- Worked with EMU output using menu options from package as well as custom plots from raw output dictionaries

*To do*

- [ ] Clean up notebooks and consolidate code into final docs on github repo
- [ ] Finish annotating code and github README to document what we did

**Comment by mjzahn** [comment]
*Summary for work completed on Oct 18, 2024*

- Cleaned up notebooks and our github repo
- Finalized powerpoint slide which is available here:

[ESPRESSO FINAL PROJECT SLIDE](https://docs.google.com/presentation/d/1KKr9KIAILeS5JWGuom2XTw4ymFWFC_LilzXHi3rYuZU/edit?usp=sharing)
